README.md

```diff
@@ -234,7 +236,7 @@ If you like this project please follow me on github & twitter and also connect o
 
```

Comment:
- [ ] [optional] @KUBAHO3 it would be nice if you consider updating the instruction to run tests in your getting started section, the current examples runs tests for linter check , I would advice removing them and replace with ` npm run test `

Reply:
Yes I have fixed that as well, Thank you